packages/genesys-spark-components/src/components/stable/gux-dismiss-button/gux-dismiss-button.tsx

```diff
@@ -21,6 +24,9 @@ export class GuxDismissButton {
   @Prop()
   position: GuxDismissButtonPosition = 'absolute';
 
+  @Prop()
+  size: GuxDismissButtonSize = 'medium';
```

Comment:
Just curious if we will only have ever 2 values for this size: `medium` or `small`, or we will have `large` as well in the future?